android/app/src/main/java/com/microsoft/codepush/react/CodePushPackage.java

```diff
@@ -153,12 +153,13 @@ public WritableMap getPackage(String packageHash) {
 
     public void downloadPackage(Context applicationContext, ReadableMap updatePackage,
                                 DownloadProgressCallback progressCallback) throws IOException {
-
-        String newPackageFolderPath = getPackageFolderPath(CodePushUtils.tryGetString(updatePackage, PACKAGE_HASH_KEY));
-        if (FileUtils.fileAtPathExists(newPackageFolderPath)) {
+        String newUpdateHash = CodePushUtils.tryGetString(updatePackage, PACKAGE_HASH_KEY);
+        String newUpdateFolderPath = getPackageFolderPath(newUpdateHash);
```

Comment:
I notice you're renaming "package" to "update" here and there.  You could also rename "updatePackage", "PACKAGE_HASH_KEY", "PACKAGE_FILE_NAME" and "getPackageFolderPath."

There are probably more symbols that could be renamed, but these are what I saw right here.

It need not be done with this pull request.